tests/30-interop.js

```diff
@@ -18,6 +17,11 @@ const {
 } = filterByTag({tags: ['VC-API'], property: 'verifiers'});
```

Comment:
```suggestion
} = filterByTag({tags: ['VC-API'], property: 'verifiers'});
```

I thought we lower-cased tag names?

Reply:
Made an issue for it. I also need to change the zcap secret from`CLIENT_SECRET_DB` `SECRET_SEED_DB` then we can use `CLIENT_SECRET_DB` for our oauth2 DB issuers.